src/builtin/builtin_alias.c

```diff
@@ -0,0 +1,92 @@
+/*
+** EPITECH PROJECT, 2020
+** buitlin_alias.c
+** File description:
+** builtin_alias
+*/
+
+#include "shell.h"
+#include "builtin.h"
+#include <utility.h>
+#include <stdlib.h>
+#include <string.h>
+#include <stdio.h>
+
+int builtin_alias(char **argv, env_t *env)
+{
+    if (!argv[1])
+        print_aliases(env->alias);
+    if (argv[1])
+        return(add_alias(&(env->alias), argv[1], &argv[2]));
+    return (0);
+}
+
+int add_alias(alias_t **list, char *alias, char **command)
+{
+    alias_t *tmp = *list;
+    alias_t *elem = NULL;
+
+    if (!command[0])
+        return (0);
+    elem = malloc(sizeof(alias_t));
+    if (!elem)
+        return (1);
+    *elem = (alias_t){1, alias, concatenate(command), NULL};
+    if (!elem->command)
+        return (1);
```

Comment:
if you want to keep an int return in the function errors are usually less than 0 (ex: -1) otherwise use a bool type return